packages/react-native/cli.js

```diff
@@ -23,6 +23,14 @@ const deprecated = () => {
   );
 };
 
+function findCommunityCli(startDir = process.cwd()) {
```

Comment:
What is `process.cwd()` in this context? It seems fragile to assume that's where the dependency is declared.

Reply:
It's where the `react-native` command is being executed from, which should be project root. I am not familiar with use cases where it's being executed elsewhere.